src/hardware/input/intel8042.cpp

```diff
@@ -423,7 +423,7 @@ static void restart_kbd_disabled_timer()
 	if (kbd_disabled_timer_running) {
 		PIC_RemoveEvents(kbd_disabled_timer_handler);
 	}
-	PIC_AddEvent(kbd_disabled_timer_handler, kbd_disabled_duration_ms);
+	PIC_AddEvent(kbd_disabled_timer_handler, KbdDisabledDuration_ms);
```

Comment:
I don't think we should mix camel and underscore cases like this. Existing example from `mixer.cpp`, I think this should be the style:

```
constexpr auto EnvelopeMaxExpansionOverMs = 15u;
constexpr auto MaxPrebufferMs = 100;
```